.github/workflows/ci.yaml

```diff
@@ -0,0 +1,63 @@
+name: Actions
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  lint:
```

Comment:
For lint we should use the project node version from nvmrc, then use the matrix for test to exercise multiple node versions